wiki/Tournaments/Official_support/en.md

```diff
@@ -204,7 +206,7 @@ Flooding the game with profile badges dilutes the prestige of the prize for ever
 
 ### Can I request a badge for a beatmap contest or other type of competition? {#contest-support}
 
-For beatmap contests, yes, just follow the same rules as above. We highly recommend using [Featured Artists](https://osu.ppy.sh/beatmaps/artists) for such contests and will flagrantly give preferential treatment to those who do.
+For beatmap contests, yes! See [Official contest support](/wiki/Contests/Official_support) for details.
```

Comment:
Rather than splitting this up here, I think it would be easier to just point to the contest support page in general. This results in one less place to update if guidelines for new types of contests are added in the future, or the guidelines for non beatmap contests get updated